library/src/main/java/com/novoda/downloadmanager/lib/Request.java

```diff
@@ -246,6 +247,15 @@ public Request allowTarUpdates() {
     }
 
     /**
+     * When a ETag header is present, the application should check the integrity of the
+     * downloaded file, otherwise the current download won't be able to be resumed
```

Comment:
this says

>the application should check the integrity of the downloaded file

what does that mean? does the client need to check the md5 of the downloaded file or something like this?